src/modules/Builder/features/Activities/Activities.const.ts

```diff
@@ -5,3 +5,6 @@ export const EditablePerformanceTasks: string[] = [
   EditablePerformanceTasksType.Gyroscope,
   EditablePerformanceTasksType.Touch,
 ];
+
+export const withoutNumberRegex = /^(.*?)\s*\(\d+\)$/;
```

Comment:
could you please left a couple of examples for whicj strings this regexp will be applied?